apps/passport-server/test/generic-issuance/pipelines/lemonade/lemonadePipeline.spec.ts

```diff
@@ -124,8 +133,10 @@ describe("generic issuance - LemonadePipeline", function () {
     );
 
     await overrideEnvironment({
+      ...testingEnv,
       GENERIC_ISSUANCE_ZUPASS_PUBLIC_KEY: zupassPublicKey,
-      ...testingEnv
+      STYTCH_BYPASS: "true",
+      NODE_ENV: "test"
```

Comment:
is there a good reason to add this type of `NODE_ENV`?

Reply:
The default is for `NODE_ENV` to be set to `"production"`. However, the Stytch code will throw an exception if the environment is `"production"` and there is no Stytch configuration, so it must be set to something else.